.github/workflows/ci-sbt.yml

```diff
@@ -0,0 +1,49 @@
+# Find full documentation here https://docs.github.com/en/actions/learn-github-actions/workflow-syntax-for-github-actions
+name: CI
+
+on:
+  pull_request:
```

Comment:
Do we need to do anything to be cautious about building PRs from forks, here? I think there are a few interacting settings in GHA for this, so it'd be worth making sure we're 100% sure how this behaves for public repos, and that we're all consistently using a safe approach.

Reply:
From https://docs.github.com/en/actions/learn-github-actions/events-that-trigger-workflows#pull_request

> Workflows don't run on forked repositories by default. You must enable GitHub Actions in the Actions tab of the forked repository.
> 
> When a first-time contributor submits a pull request to a public repository, a maintainer with write access may need to approve running workflows on the pull request.

I think this is safe.

More here https://docs.github.com/en/actions/managing-workflow-runs/approving-workflow-runs-from-public-forks.